src/schemas/movies.py

```
@@ -0,0 +1,154 @@
+    movies: List[MovieListItemSchema]
+    prev_page: Optional[str]
+    next_page: Optional[str]
+    total_pages: int
+    total_items: int
+
+    model_config = {
+        "from_attributes": True
+    }
+
+
+class MovieCreateSchema(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)